mgmtsystem_nonconformity_quality_control_oca/tests/test_inspection.py

```diff
@@ -0,0 +1,61 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+
+from odoo.tests import common
+
+logger = logging.getLogger(__name__)
```

Comment:
```suggestion
from odoo.tests import common
```

Seems `logger` is not used

Reply:
Yes, I forget to delete it